app/src/main/res/xml/appfilter.xml

```diff
@@ -1569,6 +1572,9 @@
     <!-- Mein Vodafone -->
     <item component="ComponentInfo{com.appseleration.android.selfcare/com.vis.meinvodafone.view.activity.launch.VfLaunchBaseActivity}" drawable="meinvodafone" />
 
+    <!-- Mes Comptes - LCL -->
+    <item component="ComponentInfo{fr.lcl.android.customerarea/fr.lcl.android.customerarea.activities.notconnected.SplashScreenActivity}" drawable="mescomptes-lcl" />
+
```

Comment:
Could you rename the drawable without a dash? I don't think Android likes dashes in resource names.

Reply:
Didn't know about that sorry, just pushed a fix for it, thanks!